etc/docker/elyra/Dockerfile

```
@@ -24,7 +24,7 @@ ARG TAG="dev"
 COPY requirements.txt .
 
 # Install Elyra and custom requirements
-RUN python3 -m pip install --quiet --no-cache-dir --use-deprecated=legacy-resolver \
+RUN python3 -m pip install --quiet --no-cache-dir \
     elyra[all]=="$TAG" \
```

Could we go ahead and join these lines since that increases readability and there's (now) no reason to split them?
```suggestion
RUN python3 -m pip install --quiet --no-cache-dir elyra[all]=="$TAG" \
```